docker/clickhouse/local/init-db.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+set -e
+
+clickhouse client -n <<'EOSQL'
+CREATE TABLE default.otel_logs
```

Comment:
Pretty sure these statements will fail if the user has started running the OTEL collector against the target DB. The exporter creates the initial schema if one doesn't exist.

I think we need to add `IF NOT EXISTS` to these create statements. We probably also need to have a corresponding `ALTER TABLE IF EXISTS` block for each of these tables as well.

Reply:
You are right. I'm still working on finalizing the PR and agree we will need to add `IF NOT EXISTS` for creating schema. Although I think the script would be called before the otelcol